Makefile

```diff
@@ -46,6 +46,7 @@ DESC_FILE=$(CURDIR)/description
 BUILD_DEPS=vrt oUnit
 DEPS=core async sexplib core_extended
 
+SEMVER:=$(shell vrt prj semver)
```

Comment:
This makes afiniate/afin depend on afiniate/vrt, while vrt also depends on afiniate/afin